db/hints/manager.cc

```diff
@@ -611,11 +612,19 @@ future<> manager::change_host_filter(host_filter filter) {
             });
         });
     } catch (...) {
+        const sstring exception_message = eptr
+                ? seastar::format("{} + {}", eptr, std::current_exception())
+                : seastar::format("{}", std::current_exception);
```

Comment:
`std::current_exception` is a function, no? I don't think you want to print a function here (what does `seastar::format` even do in this case??)